### PR TITLE
Temporary declaration props type

### PR DESCRIPTION
**Pull request by Il9**
#45 

I think we will need some logic that convert XMLDOM to model.
I will temporarily change the Characteristics type of the model to the attributes type.

### CODE REVIEW

File: packages/model/src/qti2_2.ts [hunk 1]
```diff
@@ -3356,7 +3356,7 @@ export type ToleranceList = (string | Double)[];
  * # 9.2.1
  * https://www.imsglobal.org/question/qtiv2p2p4/QTIv2p2p4-ASI-InformationModelv1p0/imsqtiv2p2p4_asi_v1p0_InfoModelv1p0.html#TabEnumeratedListClass_DataModel_View
  */
-export interface View {}
+export type View = 'author' | 'candidate' | 'proctor' | 'scorer' | 'testConstructor' | 'tutor';
```
Comment (disjukr) [inline packages/model/src/qti2_2.ts:3359]: Please do not edit this file by hand.
And this type is not the type for this property.

File: packages/react/src/types/component.ts [hunk 2]
```diff
@@ -1 +1 @@
-export type Props = Record<string, string | number | boolean | undefined>;
+export type Props = Record<string, unknown>;
```
Comment (ragingwind) [inline packages/react/src/types/component.ts:1]: Why you use unknown?

Comment (Il9) [reply packages/react/src/types/component.ts:1]: At first, the type didn't match, so I put it back in, but it works just fine.

File: packages/model/src/qti2_2.ts (hunk 1 above)
Comment (Il9) [reply packages/model/src/qti2_2.ts]: I changed it to declare it in a place where it is temporarily used.

Comment (Il9) [reply packages/model/src/qti2_2.ts]: https://github.com/riiid/qtikit/pull/49/commits/cc4e83af923c9e3e3dcba2576c61b3e8dd303fd3
https://github.com/riiid/qtikit/pull/49/commits/7405a4fa7c42df78debfe91128670299332ae65f

File: packages/react/src/types/component.ts (hunk 2 above)
Comment (Il9) [reply packages/react/src/types/component.ts]: https://github.com/riiid/qtikit/pull/49/commits/de71c5d37cbf3d6f450851f1e91acf48c7a68302

File: packages/react/src/components/RubricBlock.tsx [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-import {RubricBlockCharacteristics as RubricBlockProps} from '@qtikit/model/lib/qti2_2';
+import React from 'react';
+import {BaseSequenceXBaseCharacteristics, RubricBlockCharacteristics} from '@qtikit/model/lib/qti2_2';
 
+import {QtiModelProps} from '../types/props';
 import {classNameForComponent} from '../utils/style';
 
-const RubricBlock: React.FC<RubricBlockProps | any> = props => (
+export type RubricBlockProps = QtiModelProps<BaseSequenceXBaseCharacteristics, RubricBlockCharacteristics> & {
+  view: 'author' | 'candidate' | 'proctor' | 'scorer' | 'testConstructor' | 'tutor';
```
Comment (disjukr) [inline packages/react/src/components/RubricBlock.tsx:8]:
```suggestion
  view: ('author' | 'candidate' | 'proctor' | 'scorer' | 'testConstructor' | 'tutor')[];
```

Comment (Il9) [reply packages/react/src/components/RubricBlock.tsx]: https://github.com/riiid/qtikit/pull/49/commits/2c45565cf70af177fabc5d14fe6145c2fa64b188

**Review by ragingwind — APPROVED** [review]

**Review by disjukr — APPROVED** [review]

**Pull request merged**